idl/response/foxsi_get_spectral_image.pro

```diff
@@ -0,0 +1,378 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;FUNCTION:      "foxsi_get_spectral_image"
```

Comment:
Ok, I see where the confusion is coming from. This function is the main one which processes an input data set. get_foxsi_source_map_spectrum is currently the function that generates the default spectrum. I'll change the names all round to make this clearer.